python/tests/backends/test_IQM.py

```diff
@@ -44,9 +44,15 @@ def startUpMockServer():
 
     # Set the targeted QPU
     os.environ["IQM_TOKENS_FILE"] = tmp_tokens_file.name
-    cudaq.set_target("iqm",
-                     url="http://localhost:{}".format(port),
-                     **{"qpu-architecture": "Apollo"})
+    with os.popen("git rev-parse --show-toplevel") as f:
```

Comment:
> This will only work in a git repo, right?

That's correct, and I believe it's a valid assumption, but I'm also open to other ideas of how to get relative paths to other directories if you have any suggestions.

Reply:
> This will only work in a git repo, right?

I added [88828b6](https://github.com/NVIDIA/cuda-quantum/pull/1084/commits/88828b64a5233118b618dce3e52a6001e724fb69) to fallback on no provided `mapping_file` if its running outside a git repo.